elasticsearch/linux_distributions/opendistro-tar-install.sh

```diff
@@ -21,6 +21,15 @@ bash $ES_HOME/plugins/opendistro_security/tools/install_demo_configuration.sh -y
 ##Perf Plugin
 chmod 755 $ES_HOME/plugins/opendistro_performance_analyzer/pa_bin/performance-analyzer-agent
 chmod -R 755 /dev/shm
+chmod a+rw /tmp
+chown -R elasticsearch $ES_HOME/performance-analyzer-rca
+chown elasticsearch $ES_HOME/data/rca_enabled.conf
+mkdir -p "/var/lib/elasticsearch"
+echo 'true' > /var/lib/elasticsearch/performance_analyzer_enabled.conf
+echo 'true' > /var/lib/elasticsearch/rca_enabled.conf
```

Comment:
I think it's better to use a relative file path. could we put these configuration into $ES_HOME/performance-analyzer-rca?

Reply:
btw, I don't see this change in docker image. is it required?

Reply:
removed these, can be addressed in user documentation